### DISCUSSION TITLE
Why can't I adjust the height of the button

### DISCUSSION THREAD

**Discussion by chuckding25** [israel-dryer/ttkbootstrap#242] [Q&A] [answered]
I  can only adjust the width of the button
when I change the height
here is the wrong
`_tkinter.TclError: unknown option "-height"`

**Comment by israel-dryer** [comment, marked as answer]
Technically, there is no width and height setting of the button. You can specify the [width of the text inside](https://tcl.tk/man/tcl8.6/TkCmd/ttk_widget.htm#M-width) the button.

You can use the `padding` parameter to add padding to the inside of the button and this will let you make it bigger vertically and horizontally.

**Comment by israel-dryer** [comment]
There is no option to adjust height. You can adjust the padx in the pack or
grid manager to achieve a similar result.
